docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -24,6 +24,10 @@ spec:
 
 Currently only Secrets are supported.
 
+== Trust Kubernetes CA
+
+The cert for the Kubernetes CA is added automatically to your Keycloak pods managed by the operator.
```

Comment:
The Kubernetes people use capital letters for all Kubernetes entities, so I suppose we should do the same. Instead of `Keycloak`, always use the attribute.

```suggestion
The cert for the Kubernetes CA is added automatically to your {project_name} Pods managed by the Operator.
```